### PR TITLE
#168366818 Admin can change user to mentor

### PR DESCRIPTION
**Pull request by Jajabenit250**
#### What does this PR do?
Admin should change user to mentor (API) with Postgresql Database

#### Description of Task to be completed?
- added function to change user to mentor when you are adding
- solved all feedback issue of the previous APIs and challenge in general

#### How should this be manually tested?
Clone the repository and cd into it. run npm install and after run npm start. Open postman and run
http://localhost/api/v1/user/:id but add token with Admin user  and the add required parameters in a body.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#168366818

#### User Profile before changed to mentor
User before changed to user
#### admin before changing user to Mentor
Admin before changing user to mentor

#### Changing to mentor when it is completely done
Admin after changing user to mentor

#### after changing user become mentor
User After changed to mentor

#### when you are not admin
not admin

Questions:

### CODE REVIEW

**Comment by Mirriam-Maina** [comment]
@Jajabenit250 isadmin should not be `null` by default. It should either be `true` or `false`

**Comment by Jajabenit250** [comment]
> @Jajabenit250 isadmin should not be `null` by default. It should either be `true` or `false`

Okay am going to solve the issue

**Comment by Mirriam-Maina** [comment]
This looks good. @Jajabenit250 you can go ahead and merge

File: server/middlewares/menteeChecker.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import response from '../helpers/response';
+import users from '../models';
+
+module.exports = function(req, res, next) {
```
Comment (Mirriam-Maina) [inline server/middlewares/menteeChecker.js:4]: This is `ES5` ensure your whole app is in `ES6`

Comment (Jajabenit250) [reply server/middlewares/menteeChecker.js:4]: Okay thanks I will Apply the changes to next PR

**Pull request merged**